### ISSUE TITLE
Ash dragon wanders out of range and gets stuck

### ISSUE THREAD

**Issue by Goutan** [EdenServer/community#5229]
### Checklist

- [ x] I have searched for existing issues for issues like this one. The issue has not been posted.
- [x ] I have provided reproducible steps. (No "as the title says" posts please. ***Please try to state facts only, and avoid assumptions/opinions.***)
- [ x] I have provided my client version in the details. (type /ver into your game window)
- [x ] ***I have provided proof of my claim (era videos, links to era wiki and/or forum posts)*** Lack of proof will slow down fixes and prioritization. Please provide details so that developers' research time is minimized.

![image](https://user-images.githubusercontent.com/62845606/186505494-d7c7e521-9490-4d5f-98b5-8f42a585b6ca.png)

version 30181205_0

**Comment by Steam-arch** [comment]
Maybe abit more info? Needs to be pulled first god knows where id guess?

**Comment by Goutan** [comment]
yeah sorry the report was lacking. this is the northern tunnel. it's stuck in a wall. It didn;t aggro us as we walked by. The pathing seems broken
